dashboard/clojure/src/main/openllm/components/model_selection/data.cljs

```diff
@@ -0,0 +1,40 @@
+(ns openllm.components.model-selection.data)
+
+(def models
+  {"chatglm" {:ids ["thudm/chatglm-6b"
+                    "thudm/chatglm-6b-int8"
+                    "thudm/chatglm-6b-int4"
+                    "thudm/chatglm2-6b"
+                    "thudm/chatglm2-6b-int4"]}
```

Comment:
probably best to have a script that calls `openllm models -o json` and it will dumps it to a json file of all the models available, then in clojure you can parse that json file.